libs/pmp-web/repository/feature-repository-statistics/src/lib/repository-statistics/repository-statistics.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="repository-statistics__container">
+  <pmp-header-container
+    [avatarLabel]="'Dupa'"
+    [avatarUrl]="''"
+    header="Pull requests pending"
```

Comment:
It would be better to be consistent with data bindings.
You provide avatar label like `[avatarLabel]="'string'"`, but header like `header="string"`.